src/app/game/game-result/game-result.component.ts

```diff
@@ -123,6 +131,28 @@ export class GameResultComponent implements OnInit, OnDestroy {
     this.startAnimation();
     this.translationService.loadTranslations(this.translationService.getCurrentLang()).subscribe();
   }
+  blowConfetti(): void {
+    const confettiEndTime = Date.now() + this.confettiDuration;

```

Comment:
are you referring to `confettiEndTime` any other places in the code? You have also declared this as a variable on line 54.